C# 6 tutorial: missing require in `nameof` step

https://docs.microsoft.com/en-gb/dotnet/csharp/tutorials/exploration/csharp-6?tutorial-step=8 requires an additional `require System.Collections.Generic` to work:

```
file.cs(10,9): error CS0246: The type or namespace name 'List<>' could not be found (are you missing a using directive or an assembly reference?)
file.cs(10,34): error CS0246: The type or namespace name 'List<>' could not be found (are you missing a using directive or an assembly reference?)
```


---
#### Document details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 1a3686c2-4c73-5686-0da9-abc38c74aeec
* Version Independent ID: ca995f80-9375-e9ca-b855-6e6893581e23
* Content: [Explore C# 6 - Try the new features in C# 6 interactively, using your browser](https://docs.microsoft.com/en-gb/dotnet/csharp/tutorials/exploration/csharp-6?tutorial-step=10)
* Content Source: [docs/csharp/tutorials/exploration/csharp-6.yml](https://github.com/dotnet/docs/blob/live/docs/csharp/tutorials/exploration/csharp-6.yml)
* Product: **dotnet-csharp**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

--- a/docs/csharp/tutorials/exploration/csharp-6.yml
+++ b/docs/csharp/tutorials/exploration/csharp-6.yml
@@ -188,6 +188,7 @@ items:
 
     ```csharp
     using System;
+    using System.Collections.Generic;
 
     public class Program
     {

